UI/js/not.js

```diff
@@ -0,0 +1,50 @@
+//notification
```

Comment:
Hey, great work here, 
Could you change the file name to a better descriptive and meaningful name.